packages/js/plugins/ethereum/src/__tests__/e2e.spec.ts

```diff
@@ -28,6 +29,9 @@ const contracts = {
   SimpleStorage: {
     abi: require("./contracts/SimpleStorage.ABI.json"),
     bytecode: `0x${require("./contracts/SimpleStorage.Bytecode.json").object}`,
+    abiSinglePrimitiveMethod: '[{"inputs":[],"name":"get","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"}]',
```

Comment:
would like to see a test where we use the "hand written" form of the method's signature, in addition to the JSON signature.

For example `function foo(...) returns(address[])`

Reply:
indeed, that makes lot of sense :ok_hand: